source/CCustomOpcodeSystem.cpp

```diff
@@ -738,20 +770,26 @@ namespace CLEO
 				case DT_LVAR_TEXTLABEL:
 				case DT_VAR_TEXTLABEL_ARRAY:
 				case DT_LVAR_TEXTLABEL_ARRAY:
-					return { (char*)GetScriptParamPointer(thread), 8 };
+					result.data = (char*)GetScriptParamPointer(thread);
+					result.size = 8;
+					result.needTerminator = false;
+					return result;

```

Comment:
what about DT_STRING?

```
					result.data = (char*)GetScriptParamPointer(thread);
					result.size = 16;
					result.needTerminator = false;
```

Reply:
DT_STRING is const literal and can not be written, right?

Reply:
yes

Reply:
that function is named GetStringParamWriteBuffer, so it is ok then

Reply:
right, I didn't notice it's for variables only